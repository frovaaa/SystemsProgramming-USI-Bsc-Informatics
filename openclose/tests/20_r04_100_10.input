4QqOoGXUULBbVYKkFETJQqDUYDSsdDdyLludKJjFC0cfJjXVv1RHPp4NXxnZZVJVvjT3GgWUuFfwtvzzhrxkjtefyvluuxg
YyBbUJXLJTt2FWUMB1VvWwRVOo86GCIicgXxMm5QqRNVPpMOJG0Y2YMRr5NQqPTCVvctpnmyygjomvnrvrbmuwfjlxju
CIicSsSAasUHRrORLlrIZzKGgkio0hJNn2GgOWDZYCFfcFfPDQZJLGA0agKEe8kZzYIiyljzRrN2GUuSsPpgnqdpyzdwoju
ZLIHKULJMPEXHIECcKkeJjNC8RLQqPOSWwK1ZWH8AZNYRrNKknXIMDVPpA9avZzdmixynzahwzksoplrcnihxepmjlukhilz
8WGgZ9JJPGELNnVBbFRXxrfvRQNLGFfglXNnxBGBDUEAOoFFEQXxGDVvBTJNEDdenUJLljujtbdgqeffaeudbgbnqrlegpjjzw
5ZzWwAa9JjOHQBbATUTtCcuV8QFNEUuZzJjKk3ZFOMM9FfMmLYDdICcSZFTtFTtXPpxY1Rryffzsiylmmofzenfqvtaqho
JjZSMTtZXVvxKkzms6MNISsNGOBUNIDC9EJjIieETPTtptOoXxecQqdNn7MF8fmiX7PARBbRTtrLGKkglrapxnubogninmz
NTQBIUuiMMmFfLlPVDEeVWYY8yyVNWBOAaoES8FfDdFfJNVvDdUunSLNXW4KEJJKkPpO6ojjekwxnlsjsebwnvwvdvpmbqtn
VvVv7MDKU9JSXxsjukXJWwZRZEFLCZZNSNnOR6ZHXIHhNPEQWOESLCKOoKKMNnmkkkclseowqepnixhzrosnzzclfezrzjxdm
AJjMZzWwmVSsvJjWVQqZzvwVLGOoR1JDdRR7OYSHJjMmYTSKksPZzpVKCcAEeUSTGWwMPXZVvzxpmgtsuakvtyhsyorrjrglva
