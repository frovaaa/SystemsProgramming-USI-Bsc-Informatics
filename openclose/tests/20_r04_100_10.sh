#!/bin/bash
exec "$1" "AaBbCcDdEeFfGgJjIiHhKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz" < tests/20_r04_100_10.input
