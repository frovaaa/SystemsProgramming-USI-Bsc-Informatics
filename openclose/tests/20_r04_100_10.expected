4GXUULVYFETJKFC0cfX1RH4ZZVT3tvzzhrxkjtefyvluuxg
UJXLJ2FWUMB1RV865RNVMOJG0Y2YM5myygjomvnrvrbmuwfjlxju
UH0hJ2OWDZYPDQZJLGA0agK8kljzN2nqdpyzdwoju
ZLIHKULJMPEXHINC8RLPOSK1ZWH8AZNYXIMDVA9avdmixynzahwzksoplrcnihxepmjlukhilz
8WZ9zw
59OHQATV8QFNE3ZFOMM9LYISZFFY1yffzsiylmmofzenfqvtaqho
Z6MNINGOBUNIDC9cd7MF8fmiX7xnubogninmz
NTQBMPVDVWYY8yyVNWBES8JSLNXW4KEJJO6ojjekwxnlsjsebwnvwvdvpmbqtn
7MDKU9ukXJZRZEFLCZZNSOR6rosnzzclfezrzjxdm
AVLGR1JRR7rrjrglva
